vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SpreadsheetDropdownFactory.java

```diff
@@ -44,9 +55,37 @@ public Component getCustomEditorForCell(Cell cell,
     return null;
   }
 
+  private ComboBox<AnalysisMethod> createEditorAnalysisComboBox(Spreadsheet spreadsheet,
+      int selectedCellColumnIndex, int selectedCellRowIndex) {
+    ComboBox<AnalysisMethod> editorComboBox = new ComboBox<>();
+    editorComboBox.setClassName("spreadsheet-combo-box");
+    editorComboBox.setItems(AnalysisMethod.values());
+    editorComboBox.setRenderer(createRenderer());
+    editorComboBox.addValueChangeListener(e -> {
+      if (e.isFromClient()) {
+        //We add a whitespace so the value is not auto incremented when the user drags a value
+        Cell createdCell = spreadsheet.createCell(selectedCellRowIndex, selectedCellColumnIndex,
+            e.getValue() + " ");
```

Comment:
Currently the sample registration is not possible since the validation tries to validate against the label not the set Value. I'd argue it would make sense to also show the label here.